src/matrix/room/timeline/Timeline.js

```diff
@@ -198,8 +198,10 @@ export class Timeline {
         if (!this._localEntries?.hasSubscriptions) {
             return;
         }
-        // find any local relations to this new remote event
-        for (const pee of this._localEntries) {
+        // find any local relations to these new remote events or maybe these
+        // new remote events reference one of the other new remote events we have.
+        const entryList = new ConcatList(entries, this._localEntries);
+        for (const pee of entryList) {
```

Comment:
Failure in CI, https://github.com/vector-im/hydrogen-web/runs/8104935984?check_suite_focus=true#step:7:167

```
 * src::matrix::room::timeline::Timeline::remove remote reaction ... failed
no persisted storage, database can be evicted by browser
AssertionError [ERR_ASSERTION]: 0 == -1
    at remove remote reaction (file:///home/runner/work/hydrogen-web/hydrogen-web/src/matrix/room/timeline/Timeline.js:669:20)
    at async runTests (file:///home/runner/work/hydrogen-web/hydrogen-web/node_modules/impunity/src/runner.mjs:44:5)
    at async main (file:///home/runner/work/hydrogen-web/hydrogen-web/node_modules/impunity/src/main.mjs:21:19)
```

Reply:
Fixed by adjust the value, see https://github.com/vector-im/hydrogen-web/pull/860#discussion_r959095048